cogs/zalgo.py

```diff
@@ -1,5 +1,6 @@
+'''pylint told me to put something here'''

```

Comment:
Let's keep docstrings relevant: please describe what the cog does, or remove it if you've disabled pylint for this file.